enhancements/ingress/request-timeout-configuration.md

```diff
@@ -69,6 +72,7 @@ spec:
     clientFinTimeout: "1s"
     serverTimeout: "30s"
     serverFinTimeout: "1s"
+	tunnelTimeout: "1h"
```

Comment:
The indentation is inconsistent on this line (tab instead of spaces).

Reply:
Good catch. It looks like my editor didn't do tabs-to-spaces conversion for some reason...